packages/manager/modules/pci/src/projects/project/ai/apps/apps.controller.js

```diff
@@ -1,12 +1,19 @@
+import { get } from 'lodash';
```

Comment:
`import { get } from 'lodash-es'` otherwise you're importing the whole `lodash` package instead of just what you need.

Reply:
Yes that's okay, that way you're only importing the get file, that's fine with me !